### ISSUE TITLE
Why this doesn't work

### ISSUE THREAD

**Issue by RoelN** [RoelN/Foity-Toity#1]
My theory was that all _webfonts_ in your font stack could be picked by the browser to serve as fallback font. Since Foity Toity was so small, it would be loaded before the main font, even if it was the second font in the stack.

Turns out browsers will only use _local_ fonts as fallback — not webfonts, even if they're already in the cache or sooner available than the first webfont in the stack. This could cause too many repaints (one for every time a webfont has finished loading). So it's either _all local fonts_, or _all webfonts_. 

In other words, Foity Toity will never be loaded before the first font in the stack, meaning this approach simply won't work!
